### PR TITLE
MARKET-5144: implement Navigation Bar and Footer

### PR DESCRIPTION
**Pull request by co185063**
## Description
This feature includes the navigation bar and the footer bar that should be seen on the top and bottom of every page, respectively.

## Screenshots/Screencasts
image

## Testing
Test all links and make sure they navigate to the appropriate page

## Ticket
5144

### CODE REVIEW

**Review by tm185221 — APPROVED** [review]
Looks good to me. Things work as they should. The footer seems to have less space than the Figma image but that can be a clean up ticket if it is necessary.

File: code/templates/base.html [hunk 1]
```diff
@@ -92,6 +99,39 @@
...
       });
-    </script>{% block javascript %}     {% endblock javascript %}
+    </script>
+    
+    <script src="{% static 'js/cart.js' %}" type="text/javascript"></script>
+    <script src="{% static 'js/startsession.js' %}"></script>
+    {% block javascript %}     {% endblock javascript %}
   </body>
+
+  <footer>
```
Comment (kg185155) [inline code/templates/base.html:109]: <footer> should be before the </body>

File: code/templates/base.html [hunk 2]
```diff
@@ -76,8 +85,6 @@
 
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js" integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
```
Comment (kg185155) [inline code/templates/base.html:86]: `<script>` tags should be loaded after all HTML

File: code/templates/base.html [hunk 3]
```diff
@@ -92,6 +99,39 @@
...
+    {% block javascript %}     {% endblock javascript %}
   </body>
+
+  <footer>
+    <div class="container-fluid mt-5 footer-container">
+      <div class="row pt-2 pl-md-5">
+        <div class="footer-text col-lg-6 col-md-8">
+            <div class="pt-lg-1 pt-md-4">
+                <a class="footer-link-text" href="{% url 'index' %}">
+                    <h1 class="footer-title-text">Peachtree Burger</h1>
```
Comment (kg185155) [inline code/templates/base.html:115]: use `class='h1'` instead of using `<h1>`, etc.

**Pull request merged**